.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+node_js:
+  - stable
+
+branches:
+  only:
+    - development
+    - master
+
+env:
+  global:
+    - CC_TEST_REPORTER_ID=27dd146710c70f3d67ffe244bd1effff61cbae4cce257fe1b4026439fccc25c7
```

Comment:
Should this key be secret?

Reply:
It can be. The choice is yours. You can create a secret if you want it and I use it instead of the code

Reply:
Nice, i'll do this. thx ;)